pvlib/tests/iotools/test_solcast.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
+from pvlib.iotools.solcast import (
+    get_solcast_live, get_solcast_tmy, solcast2pvlib

```

Comment:
We should test `_historical` and `_forecast` too, right?

Reply:
done.